ffmpeg/nvidia_test.go

```diff
@@ -603,6 +603,7 @@ func TestNvidia_API_AlternatingTimestamps(t *testing.T) {
 	tc := NewTranscoder()
 	idx := []int{1, 0, 3, 2}
 	for _, i := range idx {
+		// TODO this breaks with nvidia acceleration on the input!
```

Comment:
Note that this is _not_ a new problem with this PR; the old code had the same issue